Introduce Login Methods for customers (#12145)
DataDog/documentation: content/en/account_management/users/_index.md

To discover all of the roles available and how to create custom ones, see the [Role Based Access Control documentation][2].

## Edit a user's login methods

Only users with the User Access Management permission, such as users with the Datadog Admin Role, can change another user's login methods.

> Login methods for an organzation can be set by default through the Login Method interface. There you can choose to allow or disallow all your users in your organization to use Datadog Username/Password, Sign in with Google or SAML. From the User Management interface you can set an override on a per-user basis to allow a specific user to use one method or multiple methods. This is helpful in circumstances where you want all your users to use SAML but need to have a set of users who can log in with Username and Password in an emergency.

Reviewer: ```suggestion
Default login methods for an organization can be set through the Login Methods page. There you can allow or disallow all users in your organization to use a Datadog username and password, to sign in with Google, or to sign in with SAML. In User Management you can override on a per-user basis to allow a specific user to use one method or multiple methods. This is helpful in circumstances where you want all users to use SAML but need to enable a set of users to log in with username and password in an emergency.
```

To edit a user's login methods:

1. Go to the **Users** tab of **Organization Settings**.

> 2. Select the **Edit** button on the right of the user line.

Reviewer: ```suggestion
2. Click **Edit** on the right of the user line.
```

> 3. Select the toggle next to **Override Default Login Methods** to enable or disable overrides on the user.

Reviewer: ```suggestion
3. Switch the toggle beside **Override Default Login Methods** to enable or disable overrides for the user.
```

4. If enabling overrides, choose a set of login methods that the user can use to access Datadog. This can be a single option or all options that are configured for your organization.

> 5. **Save** the new settings.

Reviewer: ```suggestion
5. Click **Save**.
```

> {{< img src="account_management/users/user_login_method_update.png" alt="User Login Method update" style="width:80%;">}}

Reviewer: ```suggestion
```

> Note: Overrides can be set only to valid login methods. If you have not configured SAML you will not be able to choose that login method as an override for a user.

Reviewer: ```suggestion
**Note**: Overrides can be set only to valid login methods. If you have not configured SAML, you cannot choose that login method as an override for a user.
```
